src/main/java/bio/terra/cli/command/shared/options/CromwellPath.java

```diff
@@ -0,0 +1,12 @@
+package bio.terra.cli.command.shared.options;
```

Comment:
`--path` sounds generic. We may want to use for another command in the future. Instead of defining `shared/options/CromwellPath.java`, can you define this option in GenerateConfig.java ?

You require that CromwellPath ends in `cromwell.config`, but you don't validate it. How about passing in directory instead of file path?

```
  @CommandLine.Option(
      names = "--dir",
      defaultValue = ".",
      required = false,
      description =
          "Directory to put generated cromwell.conf in. Defaults to current directory.")
  public String dir;
```

I think it's cleaner to put default in the CommandLine.Option, rather than in code.